<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ColumnsDlg</class>
 <widget class="QDialog" name="ColumnsDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>380</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Select Process Columns</string>
  </property>
  <layout class="QVBoxLayout" name="dialogLayout" stretch="0,1,0,0">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Choose columns to be displayed</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="columnsPaneLayout" stretch="1,0,1">
     <item>
      <widget class="QListWidget" name="listInactive">
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="showDropIndicator" stdset="0">
        <bool>false</bool>
       </property>
       <property name="itemAlignment">
        <set>Qt::AlignLeading</set>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="arrowButtonsLayout">
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="buttonActivate">
         <property name="text">
          <string>-&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="buttonInactivate">
         <property name="text">
          <string>&lt;-</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="buttonUp">
         <property name="text">
          <string>Up</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="buttonDown">
         <property name="text">
          <string>Down</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QListWidget" name="listActive">
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="showDropIndicator" stdset="0">
        <bool>false</bool>
       </property>
       <property name="itemAlignment">
        <set>Qt::AlignLeading</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="dlgButtonsLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="buttonOk">
       <property name="text">
        <string>OK</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonOk</sender>
   <signal>clicked()</signal>
   <receiver>ColumnsDlg</receiver>
   <slot>onOk()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>371</x>
     <y>357</y>
    </hint>
    <hint type="destinationlabel">
     <x>249</x>
     <y>189</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonCancel</sender>
   <signal>clicked()</signal>
   <receiver>ColumnsDlg</receiver>
   <slot>onCancel()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>372</x>
     <y>358</y>
    </hint>
    <hint type="destinationlabel">
     <x>250</x>
     <y>190</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonActivate</sender>
   <signal>clicked()</signal>
   <receiver>ColumnsDlg</receiver>
   <slot>onActivate()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>373</x>
     <y>359</y>
    </hint>
    <hint type="destinationlabel">
     <x>251</x>
     <y>191</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonInactivate</sender>
   <signal>clicked()</signal>
   <receiver>ColumnsDlg</receiver>
   <slot>onInactivate()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>374</x>
     <y>360</y>
    </hint>
    <hint type="destinationlabel">
     <x>252</x>
     <y>192</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonUp</sender>
   <signal>clicked()</signal>
   <receiver>ColumnsDlg</receiver>
   <slot>onUp()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>375</x>
     <y>361</y>
    </hint>
    <hint type="destinationlabel">
     <x>253</x>
     <y>194</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonDown</sender>
   <signal>clicked()</signal>
   <receiver>ColumnsDlg</receiver>
   <slot>onDown()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>376</x>
     <y>362</y>
    </hint>
    <hint type="destinationlabel">
     <x>254</x>
     <y>195</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>listActive</sender>
   <signal>itemClicked(QListWidgetItem*)</signal>
   <receiver>ColumnsDlg</receiver>
   <slot>onActiveClicked(QListWidgetItem*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>392</x>
     <y>180</y>
    </hint>
    <hint type="destinationlabel">
     <x>249</x>
     <y>189</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>listInactive</sender>
   <signal>itemClicked(QListWidgetItem*)</signal>
   <receiver>ColumnsDlg</receiver>
   <slot>onInactiveClicked(QListWidgetItem*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>107</x>
     <y>180</y>
    </hint>
    <hint type="destinationlabel">
     <x>249</x>
     <y>189</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>listInactive</sender>
   <signal>currentItemChanged(QListWidgetItem*,QListWidgetItem*)</signal>
   <receiver>ColumnsDlg</receiver>
   <slot>onInactiveCurrentItemChanged(QListWidgetItem*,QListWidgetItem*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>107</x>
     <y>180</y>
    </hint>
    <hint type="destinationlabel">
     <x>249</x>
     <y>189</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>listActive</sender>
   <signal>currentItemChanged(QListWidgetItem*,QListWidgetItem*)</signal>
   <receiver>ColumnsDlg</receiver>
   <slot>onActiveCurrentItemChanged(QListWidgetItem*,QListWidgetItem*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>392</x>
     <y>180</y>
    </hint>
    <hint type="destinationlabel">
     <x>249</x>
     <y>189</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>listActive</sender>
   <signal>itemSelectionChanged()</signal>
   <receiver>ColumnsDlg</receiver>
   <slot>onActiveSelectionChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>392</x>
     <y>180</y>
    </hint>
    <hint type="destinationlabel">
     <x>249</x>
     <y>189</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>listInactive</sender>
   <signal>itemSelectionChanged()</signal>
   <receiver>ColumnsDlg</receiver>
   <slot>onInactiveSelectionChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>392</x>
     <y>180</y>
    </hint>
    <hint type="destinationlabel">
     <x>249</x>
     <y>189</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onOk()</slot>
  <slot>onCancel()</slot>
  <slot>onActivate()</slot>
  <slot>onInactivate()</slot>
  <slot>onUp()</slot>
  <slot>onDown()</slot>
  <slot>onInactiveClicked(QListWidgetItem*)</slot>
  <slot>onActiveClicked(QListWidgetItem*)</slot>
  <slot>onActiveCurrentItemChanged(QListWidgetItem*,QListWidgetItem*)</slot>
  <slot>onInactiveCurrentItemChanged(QListWidgetItem*,QListWidgetItem*)</slot>
  <slot>onActiveSelectionChanged()</slot>
  <slot>onInactiveSelectionChanged()</slot>
 </slots>
</ui>
